internal/adapters/consul/sync.go

```diff
@@ -182,12 +182,26 @@ func discoveryChain(gateway core.ResolvedGateway) (*api.IngressGatewayConfigEntr
 
 		if len(services) > 0 {
 			tls := &api.GatewayTLSConfig{}
-			if len(listener.Certificates) > 0 {
+
+			if listener.TLS.MinVersion != "" {
+				tls.TLSMinVersion = listener.TLS.MinVersion
+			}
```

Comment:
```suggestion
			if listener.TLS.MinVersion != "" {
				tls.TLSMinVersion = listener.TLS.MinVersion
			} else {
			        // set secure default instead of Envoy's TLS 1.0 default
			        tls.TLSMinVersion = "TLSv1_2"
			}
```